plugin.yaml

```diff
@@ -343,25 +449,43 @@ data_types:
         type: string
         default: Tcp
         required: true
+        description: TCP, UDP, or *
       frontendPort:
+        description: |
+          The external port for this endpoint.
+          Must be unique within the LoadBalancer.
         type: integer
         required: true
       backendPort:
+        description: |
+          The internal port for this endpoint.
         type: integer
         required: true
       enableFloatingIP:
+        description: Must be enabled for failover
         type: boolean
         required: false
         default: false
       idleTimeoutInMinutes:
+        description: |
+          Specifies the timeout for the Tcp idle connection.
```

Comment:
Specifies the timeout for the ~~Tcp~~ **TCP** (_or tcp, but the preference is all caps_) idle connection.